examples/jsm/loaders/SDFParser.js

```diff
@@ -0,0 +1,61 @@
+import { Vector3 } from 'three';
+
+/**
+ * Lightweight MDL Molfile/SDF parser.
+ *
+ * @param {string} text - Raw SDF or MOL text content.
+ * @returns {Object} Parsed molecule data with `atoms` and `bonds` arrays.
+ */
+function parseSDF( text ) {
```

Comment:
As suggested at the forum, I vote to move this bit out of the library into a separate repository:

https://discourse.threejs.org/t/why-is-there-a-pdb-loader-and-no-sdf-loader/82861/2

You are ignoring many SDF properties right now that might be relevant for other use cases.

Reply:
External libraries are imported like in examples like here:

https://github.com/mrdoob/three.js/blob/c6e73de1367ddd47fa02e548a9fcc06153bbefa7/examples/webgl_loader_ifc.html#L26-L28